### PR TITLE
Fix: [Android] Opening camera/gallery does not work on Android 11

### LINKED ISSUE #2531: [Android] Opening camera/gallery does not work on Android 11
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Expected Result:
Image picking should work on any native platform

## Actual Result:
Launching the camera or opening the gallery does not work on Android 11 devices

## Action Performed:
1. Have an Android SDK Level 30 device (Android 11)
2. Open a chat
2. Press the plus button (+)
3. Tap on "Add Attachment"
4. Select "Take Photo" or "Choose from Gallery"
5. Observe error message "An error occurred while selecting an attachment..."

## Workaround:
Add the attachment through "Choose Document" or add it from web or mWeb

## Platform:
Where is this issue occurring?

- [ ] Web
- [ ] iOS
- [x] Android
- [ ] Desktop App
- [ ] Mobile Web

**Version Number:** 1.0.28-0
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/12156624/115675894-b2ab8700-a357-11eb-8dd0-383adb56d042.mp4

**Expensify/Expensify Issue URL:**

view job on upwork [here](https://www.upwork.com/ab/applicants/1397626556017115136/job-details)

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

### PR DESCRIPTION
**Pull request by kidroca**
### Details
Update the image picker and document picker libraries to their latest versions
Migrate usages to match the newer version specs

### Fixed Issues

Fixes #2531

### Tests
Try for both Android 10, 11 and iOS, desktop/web implementation haven't been changed but you can test that as well

1. Open a chat
2. Press the (+) button and select "Add Attachment" 
3. Try all the 3 options - Take Photo | Gallery | Documents
  - selecting photos documents should work
  - cancelling from take photo or the other options should not produce errors
  - changing permissions to "deny" using the camera should handle the error gracefully - see the videos bellow

### QA Steps
Same as above

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

https://user-images.githubusercontent.com/12156624/121575360-f5244280-ca2f-11eb-8454-8391e18e0e26.mp4

#### Mobile Web

https://user-images.githubusercontent.com/12156624/121575767-69f77c80-ca30-11eb-82e6-875f21b5651b.mov

#### Desktop

https://user-images.githubusercontent.com/12156624/121575084-a1195e00-ca2f-11eb-9552-7274026bd4f3.mov

###### When a user have manually denied permissions
https://user-images.githubusercontent.com/12156624/121574555-0e78bf00-ca2f-11eb-9808-8d05f5f1f814.mp4

###### Selecting a file from gallery or documents
Uploading Screen Recording 2021-06-10 at 21.47.51.mov…

###### When a user have manually denied permissions
https://user-images.githubusercontent.com/12156624/121572297-ade88280-ca2c-11eb-8ebb-f8275f806fb2.mp4

###### Selecting a file from gallery or documents
https://user-images.githubusercontent.com/12156624/121572413-d3758c00-ca2c-11eb-8a01-c35b56dd6593.mp4

###### When a user have manually denied permissions
https://user-images.githubusercontent.com/12156624/121571078-3d8d3180-ca2b-11eb-9953-6fbab428ef36.mp4

###### Selecting a file from gallery or documents
https://user-images.githubusercontent.com/12156624/121570901-061e8500-ca2b-11eb-871b-64be01ebf3fa.mp4

### CODE REVIEW

**Review by kidroca — COMMENTED** [review]
Added some notes on the changes

File: src/components/AttachmentPicker/index.native.js [hunk 1]
```diff
@@ -227,7 +233,7 @@ class AttachmentPicker extends Component {
                 .then(this.pickAttachment)
                 .catch(console.error)
                 .finally(() => delete this.onModalHide),
-            10,
+            25,
```
Comment (kidroca) [inline src/components/AttachmentPicker/index.native.js:236]: Unfortunately this is still needed even after the major version update for both the image-picker and the document-picker

**Review by Gonals — COMMENTED** [review]
Looks good, but there are a few conflicts!

**Comment by kidroca** [comment]
Merged and resolved conflicts

**Comment by kidroca** [comment]
Bump, @Gonals this is ready

**Review by Gonals — APPROVED** [review]

**Pull request merged**